encoders/src/main/scala/au/csiro/pathling/encoders/datatypes/R4DataTypeMappings.scala

```diff
@@ -34,7 +34,7 @@ class R4DataTypeMappings extends DataTypeMappings {
   private val fhirPrimitiveToSparkTypes: Map[Class[_ <: IPrimitiveType[_]], DataType] =
     Map(
       classOf[MarkdownType] -> DataTypes.StringType,
-      classOf[IdType] -> DataTypes.StringType,
+      //classOf[IdType] -> DataTypes.StringType,
```

Comment:
If this is no longer needed, I think we can just delete this.

Reply:
Removed.